src/lang/en-US.ts

```diff
@@ -457,8 +461,21 @@ const AdminPage: ResolvedIntlConfig['messages'] = {
     'admin.roles.message': `These are all the ${CommonMessages['terms.permissions']} that are currently provisioned. An administrator can update them in the {docLink}.`,
     'admin.roles.message.docLink': `authorization settings`,
     'admin.roles.message.docPath': `https://go.estuary.dev/provision`,
-    'admin.accessToken': `Access Token`,
-    'admin.accessToken.message': `Copy the access token below to authenticate the CLI client.`,
+
+    'admin.cli_api.header': `Programmatic Access to Flow`,
+    'admin.cli_api.message': `Use Refresh and Access tokens to connect to Flow programmatically.`,
```

Comment:
I think I recall Olivia making the call we should not capitalize words as much as possible. So I think `Refresh` and `Access` should not be capitalized.

Reply:
This content came directly from Dave.